### PR TITLE
[v2.9] Remove PSP references

### PR DESCRIPTION
**Pull request by chiukapoor**
### Update
* Update min kubeversion to >= 1.25.0
* Remove PSP references from the chart
* Bump `rancher/system-upgrade-controller` to `v0.13.4`
### Issue
* https://github.com/rancher/rancher/issues/44922

### CODE REVIEW

**Comment by kinarashah** [comment]
@chiukapoor Could you update `rancher/system-upgrade-controller` to latest `v0.13.4` as part of bumping things for v1.29?

**Review by kinarashah — DISMISSED** [review]

**Comment by kinarashah** [comment]
@jakefhyde / @jiaqiluo could you take a look?

File: charts/rancher-k3s-upgrader/0.7.0/Chart.yaml [hunk 1]
```diff
@@ -7,4 +7,4 @@ sources:
   - "https://github.com/rancher/system-charts/blob/dev-v2.9/charts/rancher-k3s-upgrader"
 version: 0.7.0
 appVersion: v0.13.1
```
Comment (jiaqiluo) [inline charts/rancher-k3s-upgrader/0.7.0/Chart.yaml]:
```suggestion
appVersion: v0.13.4
```

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

File: charts/rancher-k3s-upgrader/0.7.0/Chart.yaml (hunk 1 above)
Comment (chiukapoor) [reply charts/rancher-k3s-upgrader/0.7.0/Chart.yaml]: Updated

**Review by kinarashah — APPROVED** [review]

**Review by jiaqiluo — APPROVED** [review]

**Pull request merged**